modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -17,24 +17,37 @@ The CLI supports both interactive shell and standard CLI modes, and works with:
 
 == Prerequisites
 
-* NodeJS, and NPM (Node's package manager) installed in your machine. +
-Go to the https://nodejs.org/en/download/[NodeJS download page] to get both resources.
-* Git version management system on your computer. +
-Go to the https://git-scm.com/downloads[Git download site] to get this resource.
+* Download and install NodeJS and NPM (Node package manager) on your system.
```

Comment:
I searched NPM for addition to the Term List and found that most references are similar to "npm (originally short for Node Package Manager)," npm is a command-line tool for interacting with a huge repository of Node.js projects.," "npm is a package manager for Node.js"....
So...it looks like we should be standardizing on "npm" (node package manager) and "Node.js." Do you disagree?